cpp/src/parquet/arrow/writer.cc

```diff
@@ -403,16 +409,31 @@ class FileWriterImpl : public FileWriter {
     }
 
     auto WriteBatch = [&](int64_t offset, int64_t size) {
+      std::vector<std::unique_ptr<ArrowColumnWriterV2>> writers;
       int column_index_start = 0;
+
       for (int i = 0; i < batch.num_columns(); i++) {
-        ChunkedArray chunkedArray(batch.column(i));
+        ChunkedArray chunkedArray{batch.column(i)};
```

Comment:
Nit: I know it's from existing code, but can you rename `chunkedArray` to `chunked_array`? Thanks.

Reply:
Good catch! Fixed the name. Please check @cyb70289